lib/preset-patterns.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   suit: {
     componentName: /[A-Z][a-zA-Z]+/,
     componentSelectors: suitSelector,
-    utilitySelectors: /^\.u(?:-[a-z][a-zA-Z0-9]+)+$/,
+    utilitySelectors: /^\.u(?:-[a-z][a-zA-Z0-9]*)+$/,
```

Comment:
The chance from `+` to `*` here and below seems unrelated ... and why is it necessary? Wouldn't you expect a warning if a class was `MyComponent--` or `.u`? Don't you expect at least one character?